base/src/com/google/idea/blaze/base/model/ExternalWorkspaceDataProvider.java

```diff
@@ -105,13 +105,11 @@ public ListenableFuture<ExternalWorkspaceData> getExternalWorkspaceData(
               .getBuildSystem()
               .getDefaultInvoker(project, context);
 
-      deps = BlazeModRunner.getInstance().getDeps(project, buildInvoker, context, importSettings.getBuildSystem(), blazeFlags);
```

Comment:
can we remove the field declaration for `deps` if it is not used?

Reply:
Ok, but I'm going to use it in later prs, just didn't have time to finish. Required for bazel 8